rspotify-http/Cargo.toml

```diff
@@ -18,20 +18,15 @@ rspotify-model = { path = "../rspotify-model", version = "0.10.0" }
 
 # Temporary until https://github.com/rust-lang/rfcs/issues/2739, for
 # `maybe_async`.
-async-trait = { version = "0.1.48", optional = true }
-base64 = "0.13.0"
-futures = { version = "0.3.8", optional = true }
-log = "0.4.11"
-maybe-async = "0.2.4"
-reqwest = { version = "0.11.0", default-features = false, features = ["json", "socks"], optional = true }
-serde_json = "1.0.57"
-thiserror = "1.0.20"
-ureq = { version = "2.0", default-features = false, features = ["json", "cookies"], optional = true }
-url = "2.2.2"
+async-trait = { version = "0.1.51", optional = true }
+log = "0.4.14"
```

Comment:
Since you have replaced `log` with `debug_assert`, is it possible to remove this crate?

Reply:
I was going to remove it but I think it's still useful to have around. I wanted to add some log statements before releasing the new version because we don't even log that much right now. But I wanted to wait for the pending PRs to be merged first.